dev/com.ibm.ws.componenttest.2.0/bnd.bnd

```diff
@@ -0,0 +1,74 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
```

Comment:
Minor: Wrong year copyright for new file.  Some other files have the same issue.